spec/migration/update-6_0_0/index-spec.ts

```diff
@@ -13,11 +13,12 @@ describe('Migration Schematic', () => {
     tree.create('/package.json', `{}`);
   });
 
-  it('adds missing dependencies', () => {
+  it('adds missing dependencies', (done) => {
```

Comment:
Is the `done` here really needed? Is there a particular reason for adding it?  It seems superfluous.

Reply:
@cartant Sorry, should have commented on that. When I added the script and ran `npm run test_no_cache`, the test failed with a timeout without having the done.